presto-native-execution/etc/config.properties

```diff
@@ -1,3 +1,3 @@
-discovery.uri=http://127.0.0.1:58215
+discovery.uri=http://localhost:8080
```

Comment:
Looks like this step needs changing as well, since, the host name is no longer matching and needs to be updating. 

> Copy that port (or the whole URL) to the discovery.uri

Reply:
Since the document says `or the whole URL` I did not change this. But will update it to be clear.

Reply:
It would be nice to update, since the option to change just the port is no longer available.

Reply:
I changed it. But it will still work where `localhost` is the same as `127.0.0.1`.

Reply:
Got it. Thank you for clarifying.